src/server/verification/verificationAPI.js

```diff
@@ -196,7 +195,10 @@ const setup = (app: Router, verifier: VerificationAPI, gunPublic: StorageAPI, st
       log.debug('sending otp:', user.loggedInAs)
 
       if (!userRec.smsValidated || hashedMobile !== savedMobile) {
-        const [, code] = await sendOTP({ mobile })
+        let code
```

Comment:
Twilio's resolved response isn't used anywhere. So let's sendOTP resolves with the code only:

```javascript
export const sendOTP = (user: UserRecord): Promise<any> => {
  const { twilioAuthID, twilioAuthToken, twilioPhoneNumber } = conf
  const { mobile } = user

  const client = Twilio(twilioAuthID, twilioAuthToken)

  const otp = generateOTP(conf.otpDigits)
  const msg = 'Your GoodDollar Verification Code Is: ' + otp

  await client.messages.create({ to: mobile, from: twilioPhoneNumber, body: msg })
  return otp
}
```

then we could make this code more readable

```
let code = null

if (['production', 'staging'].includes(conf.env)) {
  code = await sendOTP({ mobile })
}
```

Reply:
done